boards/arm/mimxrt1010_evk/init.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ #include <soc.h>
+
```

Comment:
Should include fsl_common.h instead of `soc.h`

Reply:
I'd prefer `soc.h` over `fsl_common.h`, because this include makes it more clear which definition we need: the SOC registers themselves (which are defined via `fsl_common.h`)